packages/react-components/src/components/chart/legend/table/columnDefinitions/factory.tsx

```diff
@@ -25,6 +30,7 @@ const createDataStreamColumnDefinition = (
 const createMaximumColumnDefinition =
   (): LegendTableColumnDefinitions[number] => ({
     id: 'MaxColumn',
+    sortingField: 'Maximum',
```

Comment:
What does this sorting field do? Can we add a gif of the sorting working? We've broken it in the past with changes to the columnDefinitions.

Reply:
removed this  "sortingField: 'Maximum'," for the Maximum column, sorting is handled without this. 

https://github.com/awslabs/iot-app-kit/assets/142866907/976ce718-518b-49b8-84c4-9c17158d03b1